AutoMad/storySaver.py

```diff
@@ -0,0 +1,57 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Mon Oct 21 15:44:54 2024
+
+@author: dragon
+"""
+
+import os.path
+import random
+
+import numpy as np
+import pandas as pd
+from google.auth.transport.requests import Request
+from google.oauth2.credentials import Credentials
+from google_auth_oauthlib.flow import InstalledAppFlow
+from googleapiclient.discovery import build
+from googleapiclient.errors import HttpError
+
+SCOPES = ["https://www.googleapis.com/auth/spreadsheets.readonly","https://www.googleapis.com/auth/documents"]
```

Comment:
This scope looks the same as in the other google api file. If it is indeed the same, consider having only one copy somewhere and importing it both places